README.md

```diff
@@ -146,29 +168,25 @@ some_ros_package
 ## Testing
 ### GTest
 - GTest is our primary testing tool at the moment. The ROS wiki has a quick intro to it [here](http://wiki.ros.org/gtest), and we also strongly recommend you read Google's introduction to it [here] (https://github.com/google/googletest/blob/master/googletest/docs/Primer.md), then setup and write a few example tests before you start using it with ROS.
-- Once you've setup your tests in ROS, run `catkin_make run_tests` to run them
-- To run the tests for a specific package, run `catkin_make run_tests_MY_PACKAGE_NAME`
+- Once you've setup your tests in ROS, run `catkin run_tests` to run them
+- To run the tests for a specific package, run `catkin build run_tests_MY_PACKAGE_NAME`
```

Comment:
Last one promise ;)
proper syntax for testing a specific package is :
`catkin build [package_name] --no-deps --catkin-make-args run_tests`
[Source](http://catkin-tools.readthedocs.io/en/latest/verbs/catkin_build.html#building-and-running-tests)